src/fairchem/core/datasets/lmdb_dataset.py

```diff
@@ -214,7 +212,7 @@ def get_metadata(self, num_samples: int = 100):
         }
 
 
-class SinglePointLmdbDataset(LmdbDataset[BaseData]):
+class SinglePointLmdbDataset(LmdbDataset):
```

Comment:
Given the dataset changes, this may be a good time to deprecate these....

Warning - you'll break a few tests. If you break way too many you can revert and we can do this as part of BE.